### PR TITLE
fix: broken PageHero buttons on the Layer2 page

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

- Fixes id attr on page's headers
- Fixes hidden overflow on PageHero buttons

The last fix covers the visual issue when you hover or focus the buttons.
image
Now:
image

## Related issues

Fixes https://github.com/ethereum/ethereum-org-website/pull/9980

### CODE REVIEW

File: src/pages/layer-2.tsx [hunk 1]
```diff
@@ -72,20 +73,20 @@ const SectionHeading = (props: HeadingProps) => {
...
-const ContentBox = (
-  props: HTMLAttributes<"div"> & {
-    children: ReactNode
-    isLightGrayBg?: boolean
-  }
-) => (
+const ContentBox = ({
+  isLightGrayBg,
+  ...rest
+}: BoxProps & {
```
Comment (nhsz) [inline src/pages/layer-2.tsx:79]:
```
{
  children: ReactNode
  isLightGrayBg?: boolean
}
```

should be defined as an interface

**Review by nhsz — CHANGES_REQUESTED** [review]
Thanks, asking for a small change

File: src/pages/layer-2.tsx [hunk 2]
```diff
@@ -72,20 +73,20 @@ const SectionHeading = (props: HeadingProps) => {
...
+  children: ReactNode
+  isLightGrayBg?: boolean
+}) => (
```
Comment (wackerow) [inline src/pages/layer-2.tsx:82]:
```suggestion
interface ContentBoxProps extends BoxProps {
  isLightGrayBg?: boolean
}
const ContentBox: React.FC<ContentBoxProps> = ({ isLightGrayBg, ...rest }) => (
```

File: src/pages/layer-2.tsx (hunk 1 above)
Comment (pettinarip) [reply src/pages/layer-2.tsx:79]: Agreed

File: src/pages/layer-2.tsx (hunk 2 above)
Comment (pettinarip) [reply src/pages/layer-2.tsx:82]: Thanks!

**Review by nhsz — APPROVED** [review]
LGTM!

**Review by corwintines — APPROVED** [review]

**Pull request merged**